### PR TITLE
13 feature10 add functionality to hideshow password icon

### LINKED ISSUE #13: feature#13-Add functionality to "Hide/Show password icon"
# REQUIREMENTS
The **TextInputComponent** must be able to show or hide the password.

### PR DESCRIPTION
**Pull request by Astroyayo**
# Description
* implements keyboard types to TextInputs components
* adds functionality when the type is a password, allowing to show or hide it
* The Icon change depending on the state of the password
 
  
  ## Screenshots 
image
image

### CODE REVIEW

File: flight-reservation/components/TextInputComponent/styles.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ export const styles = StyleSheet.create({
     borderWidth: 1,
     borderColor: "grey",
     height: 45,
+    paddingLeft: 12,
+    fontSize: 18,
+  },
+  inputContainer: {
+    //paddingLeft: 5,
```
Comment (wiltrouble) [inline flight-reservation/components/TextInputComponent/styles.js]: remove this comment

**Review by TonyHdez-Pi — APPROVED** [review]
Nicely done

File: flight-reservation/components/TextInputComponent/index.js [hunk 2]
```diff
@@ -1,19 +1,36 @@
-import { AntDesign } from "@expo/vector-icons";
-import React from "react";
+import { Ionicons } from "@expo/vector-icons";
+import React, { useState } from "react";
 import { styles } from "./styles";
 import { Text, TextInput, View } from "react-native";
 
-const TextInputComponent = ({ title, icon, help }) => {
```
Comment (wiltrouble) [inline flight-reservation/components/TextInputComponent/index.js:6]: Dont remove the param icon, should be available for other cases

**Review by wiltrouble — CHANGES_REQUESTED** [review]

File: flight-reservation/components/TextInputComponent/styles.js (hunk 1 above)
Comment (Astroyayo) [reply flight-reservation/components/TextInputComponent/styles.js]: addressed

File: flight-reservation/components/TextInputComponent/index.js (hunk 2 above)
Comment (Astroyayo) [reply flight-reservation/components/TextInputComponent/index.js:6]: addressed

**Review by TonyHdez-Pi — APPROVED** [review]
i like it

**Review by wiltrouble — APPROVED** [review]
looks good!

**Pull request merged**